build-tools/packages/build-tools/src/type-test-generator/main.ts

```diff
@@ -12,12 +12,75 @@ import { PackageJson } from "../common/npmPackage";
 import { buildTestCase, TestCaseTypeData } from "../typeValidator/testGeneration";
 import { getFullTypeName, getNodeTypeData, TypeData } from "../typeValidator/typeData";
 import { typeOnly } from "./compatibility";
+import { ExtractorConfig } from "@microsoft/api-extractor";
 
 // Do not check that file exists before opening:
 // Doing so is a time of use vs time of check issue so opening the file could fail anyway.
 // Do not catch error from opening file since the default behavior is fine (exits process with error showing useful message)
 const packageObject: PackageJson = readJsonSync("package.json");
+const previousPackageName = `${packageObject.name}-previous`;
+
+const previousBasePath = `./node_modules/${previousPackageName}`;
+
+const previousPackageJsonPath = `${previousBasePath}/package.json`;
+
+if (!existsSync(previousPackageJsonPath)) {
+	throw new Error(
+		`${previousPackageJsonPath} not found. You may need to install the package via pnpm install. Note that type tests logic looks specifically for a package named '${previousPackageName}'`,
+	);
+}
+const typeRollupFilePath = ExtractorConfig.tryLoadForFolder({
```

Comment:
Based on [ExtractorConfig docs](https://github.com/microsoft/rushstack/blob/main/apps/api-extractor/src/api/ExtractorConfig.ts#L380), this appears to return the api-extractor configuration details, not a type rollup path.

Reply:
I think we still need to dig into this config to get out the appropriate details, like the rollup file path(s). Note that there will likely be more than one. We're interested in the `alpha` one, if it exists. If it doesn't exist, then we presumably fall back to the `index.d.ts` file generated by tsc.